include/swift/Syntax/RawSyntax.h

```diff
@@ -434,6 +442,28 @@ class RawSyntax final
     return getLayout()[Index];
   }
 
+  /// Return the number of bytes this node takes when spelled out in the source
+  size_t getTextLength() {
```

Comment:
can we re-use `accumulateAbsolutePosition` to calculate the length?

Reply:
I'm using `accumulateAbsolutePosition` now to compute the length of a token. For all other nodes I'm still computing the value while constructing the tree because it is faster.